advanced/postgres_temp_user_strategy/README.md

```diff
@@ -0,0 +1,23 @@
+# PostgreSQL Lambda Strategy
+
+A starter template that implements a PostgreSQL Sym Flow using an AWS Lambda to manage database user permissions.
```

Comment:
Worth noting here how this is different from the other one?

Reply:
good call, done